lang/tags/operators_math.talon

```diff
@@ -19,29 +19,17 @@ op (exponent | to the power [of]):
     user.code_operator("MATH_EXPONENT")
 
 # comparison operators
-is equal:
-    user.deprecate_command("2025-01-19", "is equal", "op equal")
-    user.code_operator("MATH_EQUAL")
+is equal: user.code_operator("MATH_EQUAL")
```

Comment:
This creates a duplicate command versus [pace](https://github.com/talonhub/community/blob/d49b52e170487233ca9df5c28c278200c6956dcc/lang/tags/operators.talon#L7), no?

Reply:
I'd be inclined to remove the duplicates, otherwise this looks good to me.  I'm fine with the variants that aren't straight duplicates eg `is greater [than] or equal`,  `is (less | below) [than]` etc

Reply:
Done